docs/content/setup/installation/agent/daemonset.md

```diff
@@ -14,49 +14,111 @@ where it will get deployed on all the nodes of the cluster.
+   which is also being used by the Aperture Controller you want these Agents to connect with.
+
    ```bash
-   helm upgrade --install aperture aperture/operator -f values.yaml
+   helm upgrade --install agent aperture/aperture-agent -f values.yaml
    ```
 
-2. Alternatively, you can create the Agent Custom Resource directly on the Kubernetes cluster using the below YAML:
+3. Alternatively, you can create the Agent Custom Resource directly on the Kubernetes cluster using the below steps:
+
+   1. Create a `values.yaml` for just starting the operator and disabling the creation of Agent Custom Resource
+      and pass it with `helm upgrade`:
+
+      ```yaml
+      agent:
+        create: false
+      ```
+
+      ```bash
+      helm upgrade --install agent aperture/aperture-agent -f values.yaml
+      ```
 
-   1. Create a YAML file with below specifications:
+   2. Create a YAML file with below specifications:
 
       ```yaml
       apiVersion: fluxninja.com/v1alpha1
       kind: Agent
       metadata:
-        name: agent
+        name: Agent
       spec:
-        image:
-          registry: docker.io/fluxninja
-          repository: aperture-agent
-          tag: latest
+        etcd:
+          endpoints: ["ETCD_ENDPOINT_HERE"]
+        prometheus:
+          address: "PROMETHEUS_ADDRESS_HERE"
       ```
 
-   All the configuration parameters for the Agent Custom Resource are listed on the
-   [README](https://artifacthub.io/packages/helm/aperture/aperture-operator#aperture-custom-resource-parameters)
-   file of the Helm chart.
+      Replace the values of `ETCD_ENDPOINT_HERE` and `PROMETHEUS_ADDRESS_HERE` with the actual values of Etcd and Prometheus,
+      which is also being used by the Aperture Controller you want these Agents to connect with.
 
-   2. Apply the YAML file to Kubernetes cluster using `kubectl`
+      All the configuration parameters for the Agent Custom Resource are listed on the
+      [README](https://artifacthub.io/packages/helm/aperture/aperture-agent#agent-custom-resource-parameters)
+      file of the Helm chart.
+
+   3. Apply the YAML file to Kubernetes cluster using `kubectl`
 
       ```bash
       kubectl apply -f agent.yaml
       ```
 
-3. Once you have successfully deployed the Helm release, confirm that the
+4. If you want to modify the default parameters, you can create or update the
+   `values.yaml` file and pass it with `helm upgrade`:
+
+   ```bash
+   helm upgrade --install agent aperture/aperture-agent -f values.yaml
+   ```
+
+   A list of configurable parameters can be found in the
+   [README](https://artifacthub.io/packages/helm/aperture/aperture-agent#parameters).
+
+5. If you want to deploy the Aperture Agent into a namespace
+   other than `default`, use the `-n` flag:
+
+   ```bash
+   helm upgrade --install agent aperture/aperture-agent -n "aperture-system" --create-namespace
```

Comment:
last time I tried to install the agent helm chart without giving the namespace `aperture-system` .. It gave me a error saying `aperture-system` namespace not found  .. is the namespace hard coded  somewhere in the code ... or we can install it in default

Reply:
Did you mention the --create-namespace option with helm command? Also, if you have used the agent chart, these steps are not for that chart.